### PR TITLE
SRVKE-930: Sinkbinding page improvements

### PR DESCRIPTION
**Pull request by rh-max**
For versions 4.6+
Jira: https://issues.redhat.com/browse/SRVKE-930
Docs preview: https://deploy-preview-38797--osdocs.netlify.app/openshift-enterprise/latest/serverless/event_sources/serverless-custom-event-sources.html

### CODE REVIEW

**Comment by rh-max** [comment]
@cardil Could you PTAL?

File: modules/serverless-sinkbinding-reference.adoc [hunk 1]
```diff
@@ -204,3 +204,12 @@ This sets the `K_CE_OVERRIDES` environment variable on the `subject`:
 ----
 { "extensions": { "extra": "this is an extra attribute", "additional": "42" } }
 ----
+
+== The include label
```
Comment (sagidlow) [inline modules/serverless-sinkbinding-reference.adoc:208]:
```suggestion
== Include label
```

I don't think you need "The" here. Looking at the ones above it seems that convention is to specify the item then type of item so in this case, "Include label" or above case "Subject parameter".

File: modules/serverless-sinkbinding-reference.adoc [hunk 2]
```diff
@@ -204,3 +204,12 @@ This sets the `K_CE_OVERRIDES` environment variable on the `subject`:
...
+
+To use a SinkBinding, you need to do assign the `bindings.knative.dev/include: "true"` label to either the resource or the namespace. If the resource definition does not include the label, a cluster administrator can attach it to the namespace by running:
```
Comment (sagidlow) [inline modules/serverless-sinkbinding-reference.adoc:210]:
```suggestion
To use a SinkBinding, assign the `bindings.knative.dev/include: "true"` label to either the resource or the namespace. If the resource definition does not include the label, a cluster administrator can attach it to the namespace by running the following command:
```

Suggestion for minimalistic writing and a complete sentence.

**Review by sagidlow — COMMENTED** [review]
Small changes, otherwise LGTM.

**Review by cardil — APPROVED** [review]
/lgtm

from QE

**Pull request merged**